weeklyChallenges01/ChallengesWithTest01.cs

```diff
@@ -8,37 +8,59 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            if (num1 == num2)
```

Comment:
Good use of if/else statements but consider that this return type is type `bool` and simply putting `return num1==num2;` would return true if true and vice versa.